docs/support/tutorials/gis/eo_guide.md

```diff
@@ -0,0 +1,255 @@
+    * Spectral: the area of the electromagnetic spectrum that is observed and spectral width of each band provided
+    * Radiometric: how many values are possible for each pixel (bit-depth)
+* Costs
+    * Free: e.g. Landsat, MODIS, Sentinel, ...
+    * Non free (but might be possible to get for free/less for research): e.g. WorldView, Spot, Planet, ...
+* Preprocessing needs
+    * Raw or pre-processed data
+* User experience and knowledge
+    * RADAR/LiDAR require solid background knowledge for processing and interpretation
+    * Optical data is more easily interpreted and processed (and more pre-processed data is available)
+
+!!! default "Harmonized products"
+
+    If you plan to work with Sentinel-2 and Landsat 8, check also the 30 m harmonized Landsat 8 and Sentinel-2 product at [NASA](https://hls.gsfc.nasa.gov/).
+
+## Where can I find the data?
+
+The best place to get the data depends on your needs: Do you want to download the data into your own processing environment or do you need a processing environment close to the data? 
+
+!!! default "Other geospatial datasets"
+
+    For other geospatial vector and raster datasets, check out our [spatial dataset list on CSC research pages](https://research.csc.fi/open-gis-data#intdata3).
+
+Below is a (uncomplete) set of services, that provide download and processing capabilities:
+
+**CSC services**
+
+* Puhti
+    * [List of all available datasets in Puhti](../../../data/datasets/spatial-data-in-csc-computing-env.md)
+    * Sentinel and Landsat mosaics of Finland provided by FMI and SYKE: ```/appl/data/geo/sentinel/s2```
+    * Every CSC user has **read** access to data stored on Puhti, no need to move it, unless you need to modify it
+* Allas
+    * [List of all available geospatial datasets in Allas](../../../data/datasets/spatial-data-in-csc-computing-env.md)
+    * Sentinel-2 L2A data of Finland, where crops are grown during growing seasons of 2016-present, [usage instructions](https://a3s.fi/sentinel-readme/README.txt)
+    * Data can be directly read from Allas without download for some cases, see eg [GDAL docs](../../../apps/gdal.md) and [Allas Python examples](https://github.com/csc-training/geocomputing/blob/master/python/allas/working_with_allas_from_Python_S3.py)
+        
+    
+**Amazon Web Service (AWS)** provides worldwide data from, among others, [Sentinel-2](https://registry.opendata.aws/sentinel-2/) and [Sentinel-1 bucket](https://registry.opendata.aws/sentinel-1/), which is managed by [Sinergise](http://www.sinergise.com/). The download costs are on a "requestor pays" basis. AWS also offers the possibility for virtual machines.
+
+**Data and Information Access Services (DIAS)** (for example [ONDA](https://www.onda-dias.eu/cms/), [CREODIAS](https://creodias.eu/) or [MUNDI](https://mundiwebservices.com/)) are non-free processing platforms with data access. Data from DIAS objectstorage can also be transferred to Allas.
```

Comment:
```suggestion
**[Data and Information Access Services (DIAS)](https://www.copernicus.eu/en/access-data/dias)** are non-free processing platforms with direct data access. The services provide primarily Copernicus data.
```

Reply:
ESA SciHub and DIAS system is currently changing drastically, so this section needs to be rewritten

Reply:
I think the mention of the possibility of moving data from object storage to object storage is important. I was so happy to find out about it some time ago. It went nicely with rclone after chat with Kimmo, But maybe this should actually be mentioned somewhere in Allas docs or so; maybe it even is already(?).

Reply:
It would be good to mention that object storage to Allas is possible, but that is not specific to one service, so somewhere general.